rules/config

```diff
@@ -229,3 +229,6 @@ DEFAULT_CONTAINER_REGISTRY ?=
 # ENABLE_FIPS - support FIPS flag, if enabled, no additional config requred for the image to support FIPS
 ENABLE_FIPS_FEATURE ?= y
 ENABLE_FIPS ?= n
+
+# SONIC_SLAVE_DOCKER_DRIVER - set the sonic slave docker storage driver
+SONIC_SLAVE_DOCKER_DRIVER ?= overlay2
```

Comment:
suggest to keep the default option as vfs, and we can experiment this overlay2 in our azure pipeline more before we change this option?

Reply:
Yes, it makes sense to only enable it in azp.
Fixed, thanks.